test/helpers/minitest_spec_context.rb

```diff
@@ -0,0 +1,9 @@
+require 'minitest/spec'
```

Comment:
Yes I had thought of that. I thought it would be an `ambiguity` :stuck_out_tongue: